sp800-63a/sec6_dc.md

```diff
@@ -18,7 +18,7 @@ This section applies when the secondary authenticator is directly tied to the au
 3. The derived authenticator SHALL be valid only as long as the subscriber is authorized to hold the original authenticator.
 4. The CSP SHALL record the details of the original authenticator used as the basis for derived authenticator issuance. 
 5. The CSP SHOULD set the expiration of the derived authenticator to the expiration of the primary authenticator. There are instances where the derived authenticator need not be directly tied to the expiration of the primary authenticator as the derived authenticator can provide authentication services in its place, for example, while the expiring primary credential is being replaced.
-6. A derived authenticator MAY be issued at or below the primary authenticator IAL and/or AAL.
+6. The derived authenticator SHALL be issued at or below the primary authenticator IAL. For example, if the primary IAL is 2, the authenticator that is issued could be AAL2 or 1.
```

Comment:
This really confuses me. Why couldn't you have an AAL 3 authenticator at IAL 2? It's the converse that's the problem: you can't have an AAL 1 authenticator at IAL 2.